.vscode/launch.json

```
@@ -0,0 +1,23 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python Debugger: Module",
+            "type": "debugpy",
+            "console": "integratedTerminal",
+            "request": "launch",
+            "module": "anta.cli",
+            "args": [ "nrfu", "-c", "${workspaceFolder}/.vscode/nrfu.yml", "-i", "${workspaceFolder}/.vscode/inventory.yml", "text"],
```

```suggestion
            "args": [ "nrfu", "text"],
```
I think we should let the user run NRFU with his own environment variables. We wouldn't need `env` below and maintain the `inventory.yml` + `nrfu.yml` files.

---

Fixed by introducing management via `envFile`